internal/expr/prepare.go

```diff
@@ -183,7 +183,10 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 	}
 
 	var sql bytes.Buffer
+
+	// n counts inputs and m counts outputs.
 	var n int
+	var m int
```

Comment:
I think that `n` and `m` needing a comment to explain what they are implies that their names should be more descriptive, perhaps. How about calling them `inputIdx` and `outputIdx`?

Reply:
Fixed!